src/Management/src/EndpointBase/Metrics/ServiceCollectionExtensions.cs

```diff
@@ -55,7 +57,6 @@ public static IServiceCollection AddMetricsActuatorServices(this IServiceCollect
                 return new SteeltoeExporter(exporterOptions);
             }));
 
-            services.AddOpenTelemetryMetricsForSteeltoe();
```

Comment:
So we're going to move from configuring OTel by default to requiring explicit config (and double-checking that it wasn't added previously)... That is good for the original complaint, but (unless I'm missing something) doesn't cover the "please make it easy/automatic for me" use case. Could these methods also have some kind of "do it for me if it's MIA" option?

Reply:
I'll add a AddMetricsActuatorServicesWithOpenTelemetry that will do it for you.

Reply:
The main method we have in this area for actuators is addallactuators and the surprise was that it prevented you from adding opentelemetry later (because it seems unrelated). So I dont think we have a good recourse.